src/app/shared/services/search/search.service.ts

```diff
@@ -23,7 +23,7 @@ import { SearchState } from '../../state/search/search.state';
  * @returns The description of the search criteria.
  */
 function createSearchCriteriaDescription({ ageRange: [minAge, maxAge], gender }: SearchStateModel): string {
-  let ageDescription = `Age ${defaultTo(minAge, 0)} - ${defaultTo(maxAge, 125)}`;
+  let ageDescription = `Age ${defaultTo(minAge, 18)} - ${defaultTo(maxAge, '89+')}`;
```

Comment:
Request: Please remove the space before and after the '-' sign to better match mockup. Thanks!